cluster/aws/util.sh

```diff
@@ -805,6 +806,12 @@ function wait-for-instance-state {
       echo "Waiting for instance ${instance_id} to be ${state} (currently ${instance_state})"
       echo "Sleeping for 3 seconds..."
       sleep 3
+      attempt=$(($attempt+1))
+      if (( attempt > 200 )); then
+        echo
+        echo "Waiting for instance state failed after 10 minutes elapsed"
```

Comment:
10 minutes are too long, it should be done within 5 minutes.

Reply:
Updated.